packages/wearable_rotary/tizen/src/wearable_rotary_plugin.cc

```diff
@@ -5,18 +5,80 @@
 #include "wearable_rotary_plugin.h"
 
 #include <efl-extension/wearable/circle/efl_extension_rotary.h>
-#include <flutter/encodable_value.h>
```

Comment:
Why removed

Reply:
my mistake, I was thought there is no place to use it directly. I add this line.